src/containers/ProfilePage/ProfilePage.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+
+import {
+  createCurrentUser,
+  createListing,
+  createReview,
+  createUser,
+  fakeIntl,
+  fakeViewport,
+} from '../../util/testData';
+import { renderWithProviders as render, testingLibrary } from '../../util/testHelpers';
+
+import { ProfilePageComponent } from './ProfilePage';
```

Comment:
This is OKis, but did you consider trying to test the _actual_ ProfilePage. The benefit would be that also configs (and the relevant part of configHelper.js) would be tested at the same time.

Reply:
Is there an example page where the test targets the actual page, so I can check what the difference is?

Reply:
@SariSaar ListingCard is probably the easiest to read 
(but it uses snapshots - don't use snapshots on page component level)
[ListingCard.test.js](https://github.com/sharetribe/web-template/blob/main/src/components/ListingCard/ListingCard.test.js)

Another example could be EditListingPage.test.js:
- [Create config using util/testData.js](https://github.com/sharetribe/web-template/blob/main/src/containers/EditListingPage/EditListingPage.test.js#L153-L164)
- And then [render with created config](https://github.com/sharetribe/web-template/blob/main/src/containers/EditListingPage/EditListingPage.test.js#L261)

Reply:
The test suite has now been updated to target ProfilePage instead of ProfilePageComponent!